ansible_ai_connect/ai/api/tests/test_views.py

```diff
@@ -1359,7 +1359,6 @@ class TestFeedbackView(WisdomServiceAPITestCaseBase):
     def test_feedback_full_payload(self):
         payload = {
             "inlineSuggestion": {
-                "latency": 1000,
```

Comment:
it could have been nice to keep one of these structures with the extra keys to be sure we don't fail if a payload comes with the legacy attributes.